test/devfiles/v200/commands/ExecBadDefault.yaml

```diff
@@ -0,0 +1,18 @@
+  - id: testExec1
```

Comment:
filename is too generic. At the first look, I thought it's about default value for some field, but it's about isDefault field. So, maybe ExecBadDefaultInGroup would be a bit more self-descriptive?

Reply:
OK, will rename them.